lib/graph_presenter.rb

```diff
@@ -75,8 +75,12 @@ def word_wrap(text, line_width = 40)
   end
 
   module MethodMissingHelper
+    def respond_to_missing?(*_args)
+      true
+    end
```

Comment:
I had some problems when I tried something similar in Whitehall, although I understand very little about this aspect of Ruby... might be worth double checking the code where this has an effect.